redis/src/main/scala/zio/redis/api/Sets.scala

```diff
@@ -172,59 +179,66 @@ trait Sets {
    * @param keys Subsequent keys of sets to add
    * @return Returns the number of elements in the resulting set
    */
-  final def sUnionStore(
-    destination: String,
-    key: String,
-    keys: String*
-  ): ZIO[RedisExecutor, RedisError, Long] =
-    SUnionStore.run((destination, (key, keys.toList)))
+  final def sUnionStore[D: Schema, K: Schema](destination: D, key: K, keys: K*): ZIO[RedisExecutor, RedisError, Long] =
+    SUnionStore[D, K].run((destination, (key, keys.toList)))
 }
 
 private[redis] object Sets {
-  final val SAdd: RedisCommand[(String, (String, List[String])), Long] =
-    RedisCommand("SADD", Tuple2(StringInput, NonEmptyList(StringInput)), LongOutput)
 
-  final val SCard: RedisCommand[String, Long] = RedisCommand("SCARD", StringInput, LongOutput)
+  final def SAdd[K: Schema, M: Schema]: RedisCommand[(K, (M, List[M])), Long] =
+    RedisCommand("SADD", Tuple2(ArbitraryInput(), NonEmptyList(ArbitraryInput())), LongOutput)
```

Comment:
They're not constants anymore, so it might make sense to just inline them to the client calls above, e.g.

```scala
final def sAdd[K: Schema, M: Schema](key: K, member: M, members: M*): ZIO[RedisExecutor, RedisError, Long] = {
  val command = RedisCommand("SADD", Tuple2(ArbitraryInput(), NonEmptyList(ArbitraryInput())), LongOutput)
  
  command.run((key, (member, members.toList)))
}
```

I'm fine if you decide not to do it now to minimize the diff though.

Reply:
Done